cpp/include/cudf_test/cudf_gtest.hpp

```diff
@@ -16,86 +16,6 @@
 
 #pragma once
 
-#ifdef GTEST_INCLUDE_GTEST_GTEST_H_
```

Comment:
This is not needed now that we use a GTest version dowloaded via rapids-cmake. In addition the logic was incorrectly triggering on the devcontainer builds due to a system installed GTest causing us to break the internals of GTest